File: Coding Exercises/TypeScript/Day66/findLongerString.ts
function findLongerString(str1: string, str2: string): string {
  if (str1 === null || str2 === null) {
      throw new Error("Input strings cannot be null");
  }

  const length1: number = str1.length;
  const length2: number = str2.length;

  if (length1 > length2) {
      return str1;
  } else if (length2 > length1) {
      return str2;
  } else {
      return "Both strings are of equal length";
  }
}

File: Coding Exercises/Python/Day66/find_longer_string.py
def find_longer_string(str1, str2):
    if str1 is None or str2 is None:
        raise ValueError("Input strings cannot be None")

    length1 = len(str1)
    length2 = len(str2)

    if length1 > length2:
        return str1
    elif length2 > length1:
        return str2
    else:
        return "Both strings are of equal length"
